docs/01-overview/service-mesh/README.md

```diff
@@ -2,134 +2,4 @@
-
-# Istio limitations
-
-## Resource configuration
-
-By default, Istio resources are configured in the following matter:
-
-| Component       |          | CPU   | Memory |
-|-----------------|----------|-------|--------|
-| Proxy           | Limits   | 1000m | 1024Mi |
-| Proxy           | Requests | 10m   | 192Mi  |
-| Ingress Gateway | Limits   | 2000m | 1024Mi |
-| Ingress Gateway | Requests | 100m  | 128Mi  |
-| Pilot           | Limits   | 4000m | 2Gi    |
-| Pilot           | Requests | 100m  | 512Mi  |
-| CNI             | Limits   | 500m  | 1024Mi |
-| CNI             | Requests | 100m  | 512Mi  |
-
-## Autoscaling configuration
-
-The autoscaling configuration of the Istio components is as follows:
-
-| Component       | Min replicas | Max replicas |
-|-----------------|--------------|--------------|
-| Pilot           | 2            | 5            |
-| Ingress Gateway | 3            | 10           |
-
-The CNI component is provided as a DaemonSet, meaning that one replica is present on every node of the target cluster. Istio sidecar proxy isn't configured in terms of autoscaling as it is injected into a Pod with the [sidecar injection enabled](../../04-operation-guides/operations/smsh-01-istio-enable-sidecar-injection.md).
-
-# Useful links
-
-If you're interested in learning more about Service Mesh in Kyma, follow these links to:
-
-- Learn about the benefits of having your workload enrolled to the service mesh:
- 
-  - [Istio sidecars in Kyma and why you want them](./smsh-03-istio-sidecars-in-kyma.md)
-
-- Perform some simple and more advanced tasks:
-
-  - [Enable Istio Sidecar Proxy Injection](../../04-operation-guides/operations/smsh-01-istio-enable-sidecar-injection.md)
-
-- Troubleshoot Service Mesh-related issues when:
-
-  - You [can't access a Kyma endpoint](../../04-operation-guides/troubleshooting/service-mesh/smsh-01-503-no-access.md)
-  - [Connection refused errors](../../04-operation-guides/troubleshooting/service-mesh/smsh-02-connection-refused.md) occur
-  - [Issues with Istio sidecar injection](../../04-operation-guides/troubleshooting/service-mesh/smsh-03-istio-no-sidecar.md) come up
-  - You have an [incompatible Istio sidecar version after Kyma upgrade](../../04-operation-guides/troubleshooting/service-mesh/smsh-04-istio-sidecar-version.md)
-
-- Analyze configuration details for:
-
-   - The [Istio chart](../../05-technical-reference/00-configuration-parameters/smsh-01-istio-chart.md)
-   - The [Istio Resources chart](../../05-technical-reference/00-configuration-parameters/smsh-02-istio-resources-chart.md)
+This content has been moved to: [What is Istio in Kyma?](https://kyma-project.io/#/istio/user/00-overview/README.md).
```

Comment:
I'd remove the question mark or the period from here as they render next to each other (?.).